test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -373,7 +373,7 @@
       initPods: 2000
       measurePods: 1000
 
-- name: Preemption
+- name: BasicPreemption
```

Comment:
I'd prefer to keep the old test's name the same. This would make diffing JSON results across versions easier.

Reply:
Done